x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/manifest.json

```diff
@@ -1,29 +1,29 @@
 {
   "id": "apm_transaction",
   "title": "APM",
-  "description": "Detect anomalies in transactions from your APM services.",
+  "description": "Detect anomalies in transactions from your APM services for metric data.",
   "type": "Transaction data",
   "logoFile": "logo.json",
-  "defaultIndexPattern": "apm-*-transaction",
+  "defaultIndexPattern": "apm-*-metric-*",
   "query": {
     "bool": {
       "filter": [
-        { "term": { "processor.event": "transaction" } },
-        { "exists": { "field": "transaction.duration" } }
+        { "term": { "processor.event": "metric" } },
+        { "exists": { "field": "transaction.duration.histogram" } }
```

Comment:
These should be removed. The fact that we are doing a composite aggregation on `transaction.duration.histogram` gets us this for free.

Reply:
Having a `query` in the module `manifest.json` ensures we see a card in the ML job wizard when the index pattern the users selects in the wizard returns docs which match the query. It is not used in any way in the actual datafeed config. So I'd want to leave a `query` in the manifest so that users can create the jobs from within the ML app, as well as from within APM.

![image](https://user-images.githubusercontent.com/7405507/133110772-e91d7e3d-6a85-4fc7-a760-b7a33f940715.png)

Reply:
Oh, I see, this is to just detect the dataformatting. 

Yes, that is probably just fine. My concern is by default we shouldn't use a query in the datafeed :)